client/src/http/config.js

```diff
@@ -0,0 +1,3 @@
+export const config = {
+  baseURL: "http://localhost:3000/"
```

Comment:
What happens if we use different hosts for different env? Maybe, it would be nice to have configs for different env. Your thoughts guys.

Reply:
Then it should be some `.json` or `.js` file in the root folder. For now we only do have kinda `.env` file for this. The problem is, we cannot read variables from it unless they have stupid `VUE_APP_` prefix